src/tool/python/strings/pybase.h

```diff
@@ -750,36 +750,52 @@ namespace py
         }
     };
 
+    struct pyobj_ptr_traits
+    {
+        using type = PyObject*;
```

Comment:
>PyObject* [](start = 21, length = 9)

consider typedef-ing a PyObjectPtr and using it throughout this header - everywhere you pass and return raw PyObject* pointers, to make things exception safe.

Reply:
Done. Also cleaned up the way WinRT delegates and async complete callback capture and clean up python state